[2021-12-29 11:57:35,170] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_waterford.scrap_data scheduled__2021-12-28T10:55:00+00:00 [queued]>
[2021-12-29 11:57:36,042] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_waterford.scrap_data scheduled__2021-12-28T10:55:00+00:00 [queued]>
[2021-12-29 11:57:36,068] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-29 11:57:36,079] {taskinstance.py:1242} INFO - Starting attempt 1 of 4
[2021-12-29 11:57:36,105] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-29 11:57:36,491] {taskinstance.py:1262} INFO - Executing <Task(SparkSubmitOperator): scrap_data> on 2021-12-28 10:55:00+00:00
[2021-12-29 11:57:36,974] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'scraping_daft_waterford', 'scrap_data', 'scheduled__2021-12-28T10:55:00+00:00', '--job-id', '5330', '--raw', '--subdir', 'DAGS_FOLDER/scraping_daft_waterford.py', '--cfg-path', '/tmp/tmpl4dq4kj7', '--error-file', '/tmp/tmpsuuil060']
[2021-12-29 11:57:37,092] {standard_task_runner.py:77} INFO - Job 5330: Subtask scrap_data
[2021-12-29 11:57:36,930] {standard_task_runner.py:52} INFO - Started process 24749 to run task
[2021-12-29 11:57:38,171] {logging_mixin.py:109} INFO - Running <TaskInstance: scraping_daft_waterford.scrap_data scheduled__2021-12-28T10:55:00+00:00 [running]> on host 6bdda4c19a85
[2021-12-29 11:57:39,343] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clement.liscoet@gmail.com
AIRFLOW_CTX_DAG_OWNER=clemoni
AIRFLOW_CTX_DAG_ID=scraping_daft_waterford
AIRFLOW_CTX_TASK_ID=scrap_data
AIRFLOW_CTX_EXECUTION_DATE=2021-12-28T10:55:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-28T10:55:00+00:00
[2021-12-29 11:57:39,459] {base.py:79} INFO - Using connection to: id: spark_default. Host: spark://spark:7077, Port: None, Schema: , Login: ***, Password: ***, extra: {}
[2021-12-29 11:57:39,472] {spark_submit.py:360} INFO - Spark-Submit cmd: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py waterford
[2021-12-29 11:59:40,879] {spark_submit.py:514} INFO - Using properties file: null
[2021-12-29 11:59:45,592] {spark_submit.py:514} INFO - WARNING: An illegal reflective access operation has occurred
[2021-12-29 11:59:45,604] {spark_submit.py:514} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.6/site-packages/pyspark/jars/spark-unsafe_2.12-3.2.0.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2021-12-29 11:59:45,605] {spark_submit.py:514} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2021-12-29 11:59:45,607] {spark_submit.py:514} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2021-12-29 11:59:45,609] {spark_submit.py:514} INFO - WARNING: All illegal access operations will be denied in a future release
[2021-12-29 11:59:47,007] {spark_submit.py:514} INFO - Parsed arguments:
[2021-12-29 11:59:47,008] {spark_submit.py:514} INFO - master                  spark://spark:7077
[2021-12-29 11:59:47,011] {spark_submit.py:514} INFO - deployMode              null
[2021-12-29 11:59:47,012] {spark_submit.py:514} INFO - executorMemory          null
[2021-12-29 11:59:47,013] {spark_submit.py:514} INFO - executorCores           null
[2021-12-29 11:59:47,014] {spark_submit.py:514} INFO - totalExecutorCores      null
[2021-12-29 11:59:47,015] {spark_submit.py:514} INFO - propertiesFile          null
[2021-12-29 11:59:47,017] {spark_submit.py:514} INFO - driverMemory            null
[2021-12-29 11:59:47,019] {spark_submit.py:514} INFO - driverCores             null
[2021-12-29 11:59:47,021] {spark_submit.py:514} INFO - driverExtraClassPath    null
[2021-12-29 11:59:47,022] {spark_submit.py:514} INFO - driverExtraLibraryPath  null
[2021-12-29 11:59:47,024] {spark_submit.py:514} INFO - driverExtraJavaOptions  null
[2021-12-29 11:59:47,025] {spark_submit.py:514} INFO - supervise               false
[2021-12-29 11:59:47,027] {spark_submit.py:514} INFO - queue                   null
[2021-12-29 11:59:47,028] {spark_submit.py:514} INFO - numExecutors            null
[2021-12-29 11:59:47,031] {spark_submit.py:514} INFO - files                   null
[2021-12-29 11:59:47,035] {spark_submit.py:514} INFO - pyFiles                 null
[2021-12-29 11:59:47,036] {spark_submit.py:514} INFO - archives                null
[2021-12-29 11:59:47,037] {spark_submit.py:514} INFO - mainClass               null
[2021-12-29 11:59:47,038] {spark_submit.py:514} INFO - primaryResource         file:/opt/***/jobs/get_ads_per_county.py
[2021-12-29 11:59:47,039] {spark_submit.py:514} INFO - name                    scrap data
[2021-12-29 11:59:47,040] {spark_submit.py:514} INFO - childArgs               [waterford]
[2021-12-29 11:59:47,041] {spark_submit.py:514} INFO - jars                    null
[2021-12-29 11:59:47,042] {spark_submit.py:514} INFO - packages                null
[2021-12-29 11:59:47,043] {spark_submit.py:514} INFO - packagesExclusions      null
[2021-12-29 11:59:47,044] {spark_submit.py:514} INFO - repositories            null
[2021-12-29 11:59:47,046] {spark_submit.py:514} INFO - verbose                 true
[2021-12-29 11:59:47,048] {spark_submit.py:514} INFO - 
[2021-12-29 11:59:47,049] {spark_submit.py:514} INFO - Spark properties used, including those specified through
[2021-12-29 11:59:47,050] {spark_submit.py:514} INFO - --conf and those from the properties file null:
[2021-12-29 11:59:47,051] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-29 11:59:47,052] {spark_submit.py:514} INFO - 
[2021-12-29 11:59:47,053] {spark_submit.py:514} INFO - 
[2021-12-29 11:59:58,926] {spark_submit.py:514} INFO - Main class:
[2021-12-29 11:59:58,928] {spark_submit.py:514} INFO - org.apache.spark.deploy.PythonRunner
[2021-12-29 11:59:58,933] {spark_submit.py:514} INFO - Arguments:
[2021-12-29 11:59:58,935] {spark_submit.py:514} INFO - file:/opt/***/jobs/get_ads_per_county.py
[2021-12-29 11:59:58,937] {spark_submit.py:514} INFO - null
[2021-12-29 11:59:58,940] {spark_submit.py:514} INFO - waterford
[2021-12-29 11:59:58,944] {spark_submit.py:514} INFO - --verbose
[2021-12-29 11:59:58,968] {spark_submit.py:514} INFO - Spark config:
[2021-12-29 11:59:58,970] {spark_submit.py:514} INFO - (spark.app.name,scrap data)
[2021-12-29 11:59:58,972] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-29 11:59:58,975] {spark_submit.py:514} INFO - (spark.submit.pyFiles,)
[2021-12-29 11:59:58,978] {spark_submit.py:514} INFO - (spark.submit.deployMode,client)
[2021-12-29 11:59:58,981] {spark_submit.py:514} INFO - Classpath elements:
[2021-12-29 11:59:58,983] {spark_submit.py:514} INFO - 
[2021-12-29 11:59:58,985] {spark_submit.py:514} INFO - 
[2021-12-29 11:59:58,986] {spark_submit.py:514} INFO - 
[2021-12-29 12:00:17,752] {spark_submit.py:514} INFO - Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
[2021-12-29 12:00:17,971] {spark_submit.py:514} INFO - 21/12/29 12:00:17 INFO SparkContext: Running Spark version 3.2.0
[2021-12-29 12:00:19,874] {spark_submit.py:514} INFO - 21/12/29 12:00:19 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2021-12-29 12:00:25,232] {spark_submit.py:514} INFO - 21/12/29 12:00:25 INFO ResourceUtils: ==============================================================
[2021-12-29 12:00:25,234] {spark_submit.py:514} INFO - 21/12/29 12:00:25 INFO ResourceUtils: No custom resources configured for spark.driver.
[2021-12-29 12:00:25,260] {spark_submit.py:514} INFO - 21/12/29 12:00:25 INFO ResourceUtils: ==============================================================
[2021-12-29 12:00:25,296] {spark_submit.py:514} INFO - 21/12/29 12:00:25 INFO SparkContext: Submitted application: Get Ads per county
[2021-12-29 12:00:26,508] {spark_submit.py:514} INFO - 21/12/29 12:00:26 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2021-12-29 12:00:26,939] {spark_submit.py:514} INFO - 21/12/29 12:00:26 INFO ResourceProfile: Limiting resource is cpu
[2021-12-29 12:00:26,943] {spark_submit.py:514} INFO - 21/12/29 12:00:26 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2021-12-29 12:00:27,714] {spark_submit.py:514} INFO - 21/12/29 12:00:27 INFO SecurityManager: Changing view acls to: ***
[2021-12-29 12:00:27,715] {spark_submit.py:514} INFO - 21/12/29 12:00:27 INFO SecurityManager: Changing modify acls to: ***
[2021-12-29 12:00:27,717] {spark_submit.py:514} INFO - 21/12/29 12:00:27 INFO SecurityManager: Changing view acls groups to:
[2021-12-29 12:00:27,720] {spark_submit.py:514} INFO - 21/12/29 12:00:27 INFO SecurityManager: Changing modify acls groups to:
[2021-12-29 12:00:27,732] {spark_submit.py:514} INFO - 21/12/29 12:00:27 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(***); groups with view permissions: Set(); users  with modify permissions: Set(***); groups with modify permissions: Set()
[2021-12-29 12:00:33,508] {spark_submit.py:514} INFO - 21/12/29 12:00:33 INFO Utils: Successfully started service 'sparkDriver' on port 40065.
[2021-12-29 12:00:34,738] {spark_submit.py:514} INFO - 21/12/29 12:00:34 INFO SparkEnv: Registering MapOutputTracker
[2021-12-29 12:00:35,907] {spark_submit.py:514} INFO - 21/12/29 12:00:35 INFO SparkEnv: Registering BlockManagerMaster
[2021-12-29 12:00:37,145] {spark_submit.py:514} INFO - 21/12/29 12:00:37 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2021-12-29 12:00:37,172] {spark_submit.py:514} INFO - 21/12/29 12:00:37 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2021-12-29 12:00:37,501] {spark_submit.py:514} INFO - 21/12/29 12:00:37 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2021-12-29 12:00:39,083] {spark_submit.py:514} INFO - 21/12/29 12:00:39 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-c492019b-d84c-44a0-ade4-97debe6e098a
[2021-12-29 12:00:39,380] {spark_submit.py:514} INFO - 21/12/29 12:00:39 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2021-12-29 12:00:39,822] {spark_submit.py:514} INFO - 21/12/29 12:00:39 INFO SparkEnv: Registering OutputCommitCoordinator
[2021-12-29 12:00:44,565] {spark_submit.py:514} INFO - 21/12/29 12:00:44 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2021-12-29 12:00:44,571] {spark_submit.py:514} INFO - 21/12/29 12:00:44 WARN Utils: Service 'SparkUI' could not bind on port 4041. Attempting port 4042.
[2021-12-29 12:00:44,578] {spark_submit.py:514} INFO - 21/12/29 12:00:44 WARN Utils: Service 'SparkUI' could not bind on port 4042. Attempting port 4043.
[2021-12-29 12:00:44,584] {spark_submit.py:514} INFO - 21/12/29 12:00:44 WARN Utils: Service 'SparkUI' could not bind on port 4043. Attempting port 4044.
[2021-12-29 12:00:44,586] {spark_submit.py:514} INFO - 21/12/29 12:00:44 WARN Utils: Service 'SparkUI' could not bind on port 4044. Attempting port 4045.
[2021-12-29 12:00:44,589] {spark_submit.py:514} INFO - 21/12/29 12:00:44 WARN Utils: Service 'SparkUI' could not bind on port 4045. Attempting port 4046.
[2021-12-29 12:00:44,596] {spark_submit.py:514} INFO - 21/12/29 12:00:44 WARN Utils: Service 'SparkUI' could not bind on port 4046. Attempting port 4047.
[2021-12-29 12:00:44,716] {spark_submit.py:514} INFO - 21/12/29 12:00:44 INFO Utils: Successfully started service 'SparkUI' on port 4047.
[2021-12-29 12:00:47,921] {spark_submit.py:514} INFO - 21/12/29 12:00:47 INFO SparkUI: Bound SparkUI to 0.0.0.0, and started at http://6bdda4c19a85:4047
[2021-12-29 12:01:19,703] {spark_submit.py:514} INFO - 21/12/29 12:01:19 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-29 12:01:22,139] {spark_submit.py:514} INFO - 21/12/29 12:01:22 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-29 12:01:22,194] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-29 12:01:22,201] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-29 12:01:22,206] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-29 12:01:22,211] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-29 12:01:22,220] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-29 12:01:22,225] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-29 12:01:22,229] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-29 12:01:22,235] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-29 12:01:22,240] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-29 12:01:22,245] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-29 12:01:22,249] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-29 12:01:22,252] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-29 12:01:22,257] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-29 12:01:22,263] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-29 12:01:22,270] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-29 12:01:22,278] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-29 12:01:22,287] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-29 12:01:22,292] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-29 12:01:22,296] {spark_submit.py:514} INFO - ... 4 more
[2021-12-29 12:01:22,299] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-29 12:01:22,302] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-29 12:01:22,306] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-29 12:01:22,310] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-29 12:01:22,314] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-29 12:01:22,320] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-29 12:01:22,326] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-29 12:01:22,332] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-29 12:01:22,335] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-29 12:01:22,341] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-29 12:01:22,347] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-29 12:01:22,353] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-29 12:01:22,358] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-29 12:01:22,368] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-29 12:01:22,373] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-29 12:01:22,383] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-29 12:01:22,385] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-29 12:01:22,391] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-29 12:01:22,397] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-29 12:01:22,403] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-29 12:01:22,411] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-29 12:01:22,414] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-29 12:01:22,417] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-29 12:01:22,421] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-29 12:01:22,425] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-29 12:01:22,431] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-29 12:01:22,435] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-29 12:01:22,439] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-29 12:01:22,445] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-29 12:01:22,465] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-29 12:01:22,470] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-29 12:01:22,481] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-29 12:01:22,500] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-29 12:01:22,502] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-29 12:01:22,505] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-29 12:01:22,507] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-29 12:01:22,511] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-29 12:01:22,514] {spark_submit.py:514} INFO - ... 1 more
[2021-12-29 12:01:49,098] {spark_submit.py:514} INFO - 21/12/29 12:01:49 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-29 12:02:12,552] {spark_submit.py:514} INFO - 21/12/29 12:02:12 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-29 12:02:12,899] {spark_submit.py:514} INFO - 21/12/29 12:02:12 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-29 12:02:12,905] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-29 12:02:12,906] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-29 12:02:12,908] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-29 12:02:12,911] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-29 12:02:12,915] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-29 12:02:12,917] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-29 12:02:12,920] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-29 12:02:12,921] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-29 12:02:12,922] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-29 12:02:12,924] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-29 12:02:12,928] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-29 12:02:12,929] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-29 12:02:12,930] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-29 12:02:12,931] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-29 12:02:12,934] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-29 12:02:12,935] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-29 12:02:12,939] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-29 12:02:12,940] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-29 12:02:12,942] {spark_submit.py:514} INFO - ... 4 more
[2021-12-29 12:02:12,947] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark: Name or service not known
[2021-12-29 12:02:12,950] {spark_submit.py:514} INFO - at java.base/java.net.Inet4AddressImpl.lookupAllHostAddr(Native Method)
[2021-12-29 12:02:12,961] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$PlatformNameService.lookupAllHostAddr(InetAddress.java:929)
[2021-12-29 12:02:12,970] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1519)
[2021-12-29 12:02:12,976] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$NameServiceAddresses.get(InetAddress.java:848)
[2021-12-29 12:02:12,983] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-29 12:02:12,992] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-29 12:02:12,997] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-29 12:02:13,001] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-29 12:02:13,006] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-29 12:02:13,010] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-29 12:02:13,012] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-29 12:02:13,021] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-29 12:02:13,024] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-29 12:02:13,031] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-29 12:02:13,041] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-29 12:02:13,051] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-29 12:02:13,058] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-29 12:02:13,080] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-29 12:02:13,092] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-29 12:02:13,104] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-29 12:02:13,112] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-29 12:02:13,125] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-29 12:02:13,128] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-29 12:02:13,137] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-29 12:02:13,160] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-29 12:02:13,249] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-29 12:02:13,263] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-29 12:02:13,280] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-29 12:02:13,288] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-29 12:02:13,291] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-29 12:02:13,296] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-29 12:02:13,332] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-29 12:02:13,364] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-29 12:02:13,371] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-29 12:02:13,374] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-29 12:02:13,386] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-29 12:02:13,399] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-29 12:02:13,410] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-29 12:02:13,436] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-29 12:02:13,444] {spark_submit.py:514} INFO - ... 1 more
[2021-12-29 12:02:34,859] {spark_submit.py:514} INFO - 21/12/29 12:02:34 ERROR StandaloneSchedulerBackend: Application has been killed. Reason: All masters are unresponsive! Giving up.
[2021-12-29 12:02:34,721] {spark_submit.py:514} INFO - 21/12/29 12:02:34 WARN StandaloneSchedulerBackend: Application ID is not initialized yet.
[2021-12-29 12:02:38,078] {spark_submit.py:514} INFO - 21/12/29 12:02:38 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 35091.
[2021-12-29 12:02:38,122] {spark_submit.py:514} INFO - 21/12/29 12:02:38 INFO NettyBlockTransferService: Server created on 6bdda4c19a85:35091
[2021-12-29 12:02:38,160] {spark_submit.py:514} INFO - 21/12/29 12:02:38 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2021-12-29 12:02:38,371] {spark_submit.py:514} INFO - 21/12/29 12:02:38 INFO SparkUI: Stopped Spark web UI at http://6bdda4c19a85:4047
[2021-12-29 12:02:39,024] {spark_submit.py:514} INFO - 21/12/29 12:02:39 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 6bdda4c19a85, 35091, None)
[2021-12-29 12:02:39,048] {spark_submit.py:514} INFO - 21/12/29 12:02:39 INFO StandaloneSchedulerBackend: Shutting down all executors
[2021-12-29 12:02:39,082] {spark_submit.py:514} INFO - 21/12/29 12:02:39 INFO CoarseGrainedSchedulerBackend$DriverEndpoint: Asking each executor to shut down
[2021-12-29 12:02:39,112] {spark_submit.py:514} INFO - 21/12/29 12:02:39 INFO BlockManagerMasterEndpoint: Registering block manager 6bdda4c19a85:35091 with 434.4 MiB RAM, BlockManagerId(driver, 6bdda4c19a85, 35091, None)
[2021-12-29 12:02:39,160] {spark_submit.py:514} INFO - 21/12/29 12:02:39 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 6bdda4c19a85, 35091, None)
[2021-12-29 12:02:39,177] {spark_submit.py:514} INFO - 21/12/29 12:02:39 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 6bdda4c19a85, 35091, None)
[2021-12-29 12:02:39,180] {spark_submit.py:514} INFO - 21/12/29 12:02:39 WARN StandaloneAppClient$ClientEndpoint: Drop UnregisterApplication(null) because has not yet connected to master
[2021-12-29 12:02:39,397] {spark_submit.py:514} INFO - 21/12/29 12:02:39 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2021-12-29 12:02:40,149] {spark_submit.py:514} INFO - 21/12/29 12:02:40 INFO MemoryStore: MemoryStore cleared
[2021-12-29 12:02:40,193] {spark_submit.py:514} INFO - 21/12/29 12:02:40 INFO BlockManager: BlockManager stopped
[2021-12-29 12:02:40,279] {spark_submit.py:514} INFO - 21/12/29 12:02:40 INFO BlockManagerMaster: BlockManagerMaster stopped
[2021-12-29 12:02:40,295] {spark_submit.py:514} INFO - 21/12/29 12:02:40 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2021-12-29 12:02:41,053] {spark_submit.py:514} INFO - 21/12/29 12:02:40 INFO SparkContext: Successfully stopped SparkContext
[2021-12-29 12:02:43,968] {spark_submit.py:514} INFO - 21/12/29 12:02:43 ERROR SparkContext: Error initializing SparkContext.
[2021-12-29 12:02:43,984] {spark_submit.py:514} INFO - java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2021-12-29 12:02:43,991] {spark_submit.py:514} INFO - at scala.Predef$.require(Predef.scala:281)
[2021-12-29 12:02:43,997] {spark_submit.py:514} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2021-12-29 12:02:44,006] {spark_submit.py:514} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:603)
[2021-12-29 12:02:44,020] {spark_submit.py:514} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2021-12-29 12:02:44,035] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2021-12-29 12:02:44,039] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2021-12-29 12:02:44,047] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2021-12-29 12:02:44,057] {spark_submit.py:514} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2021-12-29 12:02:44,060] {spark_submit.py:514} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2021-12-29 12:02:44,066] {spark_submit.py:514} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2021-12-29 12:02:44,070] {spark_submit.py:514} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2021-12-29 12:02:44,073] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2021-12-29 12:02:44,088] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2021-12-29 12:02:44,093] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2021-12-29 12:02:44,099] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2021-12-29 12:02:44,108] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-29 12:02:44,113] {spark_submit.py:514} INFO - 21/12/29 12:02:43 INFO SparkContext: SparkContext already stopped.
[2021-12-29 12:02:44,120] {spark_submit.py:514} INFO - Traceback (most recent call last):
[2021-12-29 12:02:44,151] {spark_submit.py:514} INFO - File "/opt/***/jobs/get_ads_per_county.py", line 13, in <module>
[2021-12-29 12:02:44,159] {spark_submit.py:514} INFO - .appName("Get Ads per county") \
[2021-12-29 12:02:44,166] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/sql/session.py", line 228, in getOrCreate
[2021-12-29 12:02:44,177] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 392, in getOrCreate
[2021-12-29 12:02:44,181] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 147, in __init__
[2021-12-29 12:02:44,187] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 209, in _do_init
[2021-12-29 12:02:44,202] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 329, in _initialize_context
[2021-12-29 12:02:44,212] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/py4j-0.10.9.2-src.zip/py4j/java_gateway.py", line 1574, in __call__
[2021-12-29 12:02:44,230] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/py4j-0.10.9.2-src.zip/py4j/protocol.py", line 328, in get_return_value
[2021-12-29 12:02:44,239] {spark_submit.py:514} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling None.org.apache.spark.api.java.JavaSparkContext.
[2021-12-29 12:02:44,267] {spark_submit.py:514} INFO - : java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2021-12-29 12:02:44,277] {spark_submit.py:514} INFO - at scala.Predef$.require(Predef.scala:281)
[2021-12-29 12:02:44,283] {spark_submit.py:514} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2021-12-29 12:02:44,320] {spark_submit.py:514} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:603)
[2021-12-29 12:02:44,352] {spark_submit.py:514} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2021-12-29 12:02:44,362] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2021-12-29 12:02:44,371] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2021-12-29 12:02:44,376] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2021-12-29 12:02:44,387] {spark_submit.py:514} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2021-12-29 12:02:44,392] {spark_submit.py:514} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2021-12-29 12:02:44,399] {spark_submit.py:514} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2021-12-29 12:02:44,411] {spark_submit.py:514} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2021-12-29 12:02:44,420] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2021-12-29 12:02:44,428] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2021-12-29 12:02:44,432] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2021-12-29 12:02:44,444] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2021-12-29 12:02:44,450] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-29 12:02:44,465] {spark_submit.py:514} INFO - 
[2021-12-29 12:02:47,454] {spark_submit.py:514} INFO - 21/12/29 12:02:47 INFO ShutdownHookManager: Shutdown hook called
[2021-12-29 12:02:47,476] {spark_submit.py:514} INFO - 21/12/29 12:02:47 INFO ShutdownHookManager: Deleting directory /tmp/spark-46a41449-4e05-4d39-a2d1-6185190fdcdb
[2021-12-29 12:02:47,929] {spark_submit.py:514} INFO - 21/12/29 12:02:47 INFO ShutdownHookManager: Deleting directory /tmp/spark-6048ebfa-5f80-4611-b87f-dc5b5b703290
[2021-12-29 12:02:55,525] {taskinstance.py:1703} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py waterford. Error code is: 1.
[2021-12-29 12:02:55,692] {taskinstance.py:1280} INFO - Marking task as UP_FOR_RETRY. dag_id=scraping_daft_waterford, task_id=scrap_data, execution_date=20211228T105500, start_date=20211229T115735, end_date=20211229T120255
[2021-12-29 12:02:56,008] {standard_task_runner.py:91} ERROR - Failed to execute job 5330 for task scrap_data
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 292, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py waterford. Error code is: 1.
[2021-12-29 12:02:56,019] {local_task_job.py:212} WARNING - State of this instance has been externally set to up_for_retry. Terminating instance.
[2021-12-29 12:02:56,224] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 24749
[2021-12-29 12:02:56,323] {process_utils.py:66} INFO - Process psutil.Process(pid=24749, status='terminated', exitcode=1, started='11:57:36') (24749) terminated with exit code 1
